pkg/skaffold/deploy/helm.go

```diff
@@ -366,24 +388,25 @@ func (h *HelmDeployer) packageChart(ctx context.Context, r latest.HelmRelease) (
 	return filepath.Join(tmp, fpath), nil
 }
 
-func (h *HelmDeployer) getReleaseInfo(ctx context.Context, release string) (*bufio.Reader, error) {
+func (h *HelmDeployer) getReleaseManifests(ctx context.Context, release string) (*bufio.Reader, error) {
 	var releaseInfo bytes.Buffer
-	if err := h.helm(ctx, &releaseInfo, false, "get", release); err != nil {
+	if err := h.helm(ctx, &releaseInfo, false, "get", "manifest", release); err != nil {
 		return nil, fmt.Errorf("error retrieving helm deployment info: %s", releaseInfo.String())
 	}
 	return bufio.NewReader(&releaseInfo), nil
 }
 
 // Retrieve info about all releases using helm get
-// Skaffold labels will be applied to each deployed k8s object
+// Skaffold labels will be applied to each deployed k8s object including k8object from
```

Comment:
nit: `Skaffold labels will be applied to each deployed k8s object, including those created from remote charts`